ElementX/Sources/Screens/DeveloperOptionsScreen/View/DeveloperOptionsScreen.swift

```diff
@@ -66,9 +66,17 @@ struct DeveloperOptionsScreen: View {
                     Text("🥳")
                         .frame(maxWidth: .infinity)
                 }
-                .buttonStyle(FormButtonStyle())
             }
-            .formSectionStyle()
+            
+            Section {
+                Button {
+                    context.send(viewAction: .clearCache)
+                } label: {
+                    Text("Clear cache")
+                        .frame(maxWidth: .infinity)
+                        .foregroundColor(.red)
+                }
```

Comment:
Not that it matters in here, but lets stick to good habbits
```suggestion
                Button(role: .destructive) {
                    context.send(viewAction: .clearCache)
                } label: {
                    Text("Clear cache")
                        .frame(maxWidth: .infinity)
                }
```